modules/enabling-team-sync-oidc.adoc

```diff
@@ -0,0 +1,210 @@
+
+. On the *Create team* pop-up:
+
+.. Optional. Add this team to a repository.
+.. Add a team member, for example, `user1`, by typing in the user's account name.
+.. Add a robot account to this team. This page provides the option to create a robot account.
+
+. Click *Next*.
+
+. On the *Review and Finish* page, review the information that you have provided and click *Review and Finish*. 
+
+. To enable team synchronization for your {productname} OIDC deployment, click *Enable Directory Sync* on the *Teams and membership* page. Note the message in the popup:
++
+[WARNING]
+====
+Please note that once team syncing is enabled, the membership of users who are already part of the team will be revoked. OIDC group will be the single source of truth. This is a non-reversible action. Team's user membership from within Quay will be ready-only.
+====
+
+. In the popup box, enter the name of the group to sync membership with. Then, click *Enable Sync*. 
+
+. You are returned to the *Teams and membership* page. Note that users of this team are removed and are re-added upon logging back in. At this stage, only the robot account is still part of the team.
++
+A banner at the top of the page confirms that the team is synced: 
++
+[source,text]
+----
+This team is synchronized with a group in OIDC and its user membership is therefore read-only.
+----
++
+By clicking the *Directory Synchronization Config* accordion, the OIDC group that your deployment is synchronized with appears.
+
+. Log out of your {productname} registry and continue on to the verification steps.
+
+.Verification
+
+Use the following verification procedure to ensure that `user1` appears as a member of the team.
+
+. Log back in to your {productname} registry.
+
+. Click *Organizations* -> *test-org* -> *test-team* *Teams and memberships*. `user1` now appears as a team member for this team.
+
+.Verification
+
+Use the following procedure to remove `user1` from a group via your OIDC provider, and subsequently remove them from the team on {productname}.
+
+. Navigate to your OIDC provider's administration console.
+
+. Navigate to the *Users* page of your OIDC provider. The name of this page varies depending on your provider.
+
+. Click the name of the user associated with {productname}, for example, `user1`.
+
+. Remove the user from group in the configured identity provider.
+
+. Remove, or unassign, the access permissions from the user.
+
+. Log in to your {productname} registry. 
+
+. Click *Organizations* -> *test-org* -> *test-team* *Teams and memberships*. `user1` has been removed from this team.
+
+////
+[id="setting-up-keycloak-oidc-team-sync"]
+== Setting up Keycloak for OIDC team synchronization
+
+Keycloak is an open source software product to allow single sign-on with identity and access management. It can be leveraged with {productname} as an extra layer of security for your deployment. 
+
+Use the following procedure to setup Keycloak for {productname} team synchronization. 
```

Comment:
When Quay Customer is using Azure Entra ID, former name is "Azure Active Directory", how to perform the configurations in Azure?

Reply:
@LiZhang19817 I'm actually not using this procedure at all. It is currently marked out (via //// at the top and bottom of the procedure). We decided it was best to go with something more generic that spoke to all OIDC providers, rather than providing a specific example for Keycloak.